src/util.js

```diff
@@ -83,3 +84,9 @@ export const getBaseUrl = d2 => {
     const idx = api.baseUrl.indexOf('/api');
     return idx > -1 ? api.baseUrl.slice(0, idx) : api.baseUrl;
 };
+
+export const getItemUrl = (type, id, d2) => {
```

Comment:
Would it make more sense for this function to be in the itemTypes file?

Reply:
done